application/backend/app/services/data_collect/data_collector.py

```diff
@@ -22,10 +22,24 @@ class PolicyChecker(metaclass=ABCMeta):
     @abstractmethod
     def should_collect(self, timestamp: float) -> bool:
         """
-        Checks if certain dispatched image should be collected to the project dataset
+        Determines whether a dispatched image should be collected to the project dataset.
 
-        :param timestamp: dispatched image timestamp
-        :return: True if image should be collected, False otherwise
+        This abstract method defines the interface for policy checkers to evaluate
+        collection criteria based on the image timestamp. Implementations should
+        provide specific logic for when images qualify for dataset collection.
+
+        Args:
+            timestamp: Floating-point timestamp representing when the image was
+                    dispatched, typically in seconds or milliseconds since epoch.
```

Comment:
```suggestion
            timestamp: Floating-point timestamp representing when the image was
                    dispatched, in seconds since epoch.
```